runtime/calamari/src/lib.rs

```diff
@@ -791,6 +795,19 @@ impl calamari_vesting::Config for Runtime {
     type WeightInfo = weights::calamari_vesting::SubstrateWeight<Runtime>;
 }
 
+parameter_types! {
+    pub const NameServicePalletId: PalletId = NAME_SERVICE_PALLET_ID;
+}
+
+impl pallet_name_service::Config for Runtime {
+    type RuntimeEvent = RuntimeEvent;
+    type Currency = Balances;
+    type PalletId = NameServicePalletId;
+    type RegisterWaitingPeriod = ConstU32<2>;
+    type RegisterPrice = ConstU128<{ 500 * KMA }>;
```

Comment:
This is ~75 cents at current kma/usd prices. You sure this is good ?

Reply:
Honestly Just put a price, still havent gotten a price direction from Victor. Do you have any opinion on what the price should be or atleast a range?

Reply:
Not really no, what does this cost on other chains ?

Reply:
For ETH i found this ```Yearly renewals cost $5/year for names that are 5 characters or longer. 4 character names cost $160/year, and 3 character names cost $640/year. Fees are paid in ETH. The ETH/USD exchange rate is set by the Chainlink ETH/USD oracle.```

So rtight now the price is 8.97$ for an year and you can add years when you need to

Reply:
We can put it on Calamari in this form, will update with Victor if we want to go the same way with yearly registers

Reply:
Ok.
but how did you get the `8.97$ for an year` ?

Reply:
Thats the current price right now (Ethereum Name Service or ENS price) but as I said our pricing should obviously be a lot lower